pdudaemon/listener.py

```diff
@@ -87,6 +90,25 @@ async def process_request(args, config, daemon):
             return False
         args.hostname = config["aliases"][args.alias]["hostname"]
         args.port = config["aliases"][args.alias]["port"]
+    if args.group:
+        if args.hostname or args.port or args.alias:
+            logger.error("Trying to use hostname or alias or port, when also using group")
```

Comment:
This error confused me for a while; I guess what you're trying to say is; "Ignoring  group request which also included alias, port  or hostname parameters"

Reply:
Yes for better or worse it was mostly a copy from the earlier error message, which it turns out wasn't ever being logged :)

Reply:
Would be great if you could fix this one up (and potentially the other one as well, i don't mind that being together in one PR tbh)